File: persistence/src/main/java/pl/srw/billcalculator/pojo/IPgnigPrices.java
package pl.srw.billcalculator.pojo;

import pl.srw.billcalculator.db.Prices;

/**
 * Created by Kamil Seweryn.
 */
public interface IPgnigPrices extends Prices {

    String getOplataAbonamentowa();
    String getPaliwoGazowe();
    String getDystrybucyjnaStala();
    String getDystrybucyjnaZmienna();
    String getWspolczynnikKonwersji();
}

File: persistence/src/main/java/pl/srw/billcalculator/pojo/IPgnigPrices.kt
package pl.srw.billcalculator.pojo

import pl.srw.billcalculator.db.Prices

interface IPgnigPrices : Prices {

    val oplataAbonamentowa: String
    val paliwoGazowe: String
    val dystrybucyjnaStala: String
    val dystrybucyjnaZmienna: String
    val wspolczynnikKonwersji: String
}
